java/core/src/main/java/co/worklytics/psoxy/rules/RuleSet.java

```diff
@@ -0,0 +1,7 @@
+package co.worklytics.psoxy.rules;
+
+public interface RuleSet {
```

Comment:
introduce where want to bind interfaces to values of EITHER `Rules1` or `Rules2`